coral/src/app/features/login/components/LoginForm.test.tsx

```diff
@@ -47,7 +47,7 @@ describe("Login", () => {
 
     it("shows a disabled submit button for the form", () => {
```

Comment:
text needs to be updated, too. Since this is not used, there are no other updated needed -> otherwise I think we probably should confirm that users can't submit when not all required inputs are valid.